components/RegistrationForm/RegistrationForm.js

```diff
@@ -189,6 +195,11 @@ class RegistrationForm extends Component {
               {state.errorMessage && <Alert type="error">{state.errorMessage}</Alert>}
             </div>
 
+            <p>
```

Comment:
I like that you added some empty lines for breathing space here. It's just a silly bike-shedding style comment, but could you add an empty line on 123 below the other paragraph?

Reply:
Space added on line 123.